spec/facades/foods_facade_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Facade' do
+
+  it "returns fods containing a keyword" do
+    foods = FoodsFacade.get_foods('sweet potatoes')
+
+    expect(foods).to be_an(Array)
+    expect(foods).to be_all(Food)
+  end
+
+  it "returns the count of foods containing the key word" do
+    count = FoodsFacade.count_foods('sweet potatoes')
+
+    expect(count).to eq(49676)
```

Comment:
For an assessment this is totally fine, but generally I would caution against expecting the count to be an exact value because that can easily change over time with this API, breaking your test. You could instead just make sure it's an Integer, or that it's >40,000